pkg/crc/preflight/preflight_checks_common.go

```diff
@@ -55,35 +54,21 @@ var genericPreflightChecks = [...]Check{
 }
 
 func checkBundleExtracted() error {
-	if !constants.BundleEmbedded() {
+	if present, _ := constants.BundlePresent(); !present {
 		return nil
```

Comment:
Logic seems wrong here, if the bundle is not present, returning no error without any further check does not seem like what we want?

Reply:
@cfergeau this is the setup part and here if we return any error that means we are failing the setup flow, this check is only to make sure if the bundle is present in current directory or cache one, if yes then does the bundle is extracted and if not then run the fix which do the extraction.